crate/cli/src/actions/login.rs

```diff
@@ -273,13 +273,20 @@ impl LoginState {
     }
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(untagged)]
+pub enum ExpiresIn {
+    StringTimestamp(String),
```

Comment:
I am not sure to understand what you mean. Is this a question ? What should I change ?

Reply:
- is it a prerequisite to have a string representation for this timestamp? 
- is `expires_in` field used in code?
Just asking since conversion from string to int without extra knowledge is usually unsafe

Reply:
The two types comes from the different responses that we try to deserialize, coming from different Identity providers : for some this value is a string, for others it's u64.